src/components/templates/TemplateWorkflowsContent.vue

```diff
@@ -1,68 +1,134 @@
 <template>
-  <div
-    class="flex flex-wrap content-around justify-around gap-4 mt-4"
-    data-testid="template-workflows-content"
-  >
-    <div
-      v-for="template in templates"
-      :key="template"
-      :data-testid="`template-workflow-${template}`"
+  <div class="flex h-96">
+    <Listbox
```

Comment:
Listbox allows null value on `v-model:modeValue`. Currently deselect item on that listbox will cause error.

![image](https://github.com/user-attachments/assets/717cbeaf-719f-4619-b443-edf93467eba0)